README.md

```diff
@@ -15,7 +15,7 @@ And that's it!
 
 ## Deploy via One-Click
 
-[![Deploy to Scalingo](https://cdn.scalingo.com/deploy/button.svg)](https://my.scalingo.com/deploy)
+[![Deploy to Scalingo](https://cdn.scalingo.com/deploy/button.svg)](https://my.scalingo.com/deploy?source=https://github.com/Scalingo/sample-node-express)
```

Comment:
The `source` argument is not mandatory. It should work without it. Did you faced any issue using the one-click deploy here?

Reply:
Yep, unfortunately the link does not work without the source parameter.